src/assets/css/material/_main.scss

```diff
@@ -508,7 +508,7 @@ mat-chip-list {
 
 // Tooltips.
 .mat-tooltip {
-  font-size: $font-size-caption;
+  font-size: $font-size-caption !important;
```

Comment:
Can we avoid using `!important` by using more specific selector? I mean something like:

```css
.a {
  .b {
    font-size: 10px;
  }
}
```